data/json/items/toy.json

```diff
@@ -0,0 +1,131 @@
+[
+  {
+    "type": "GENERIC",
+    "id": "toy_plush",
```

Comment:
I don't see why this needs to be a separate item instead of just using teddy and not having to adjust all the spawns and add a migration.

Reply:
I mainly did so for the sake of ids not being confusing in the future, honestly. Especially since if I wanted to have the teddy bear actually spawn I'd have to make a variant ``teddy_bear_teddy`` and that's some real spaghetti. I'd rather cut confusion at the core than add more to it, honestly.

Reply:
The content of ids doesn't really matter very much, and changing ids like this means anything out of repo that uses this also needs to change, so I would strongly encourage against it.

Changing teddy bear everywhere to be a different but identical item adds more confusion to me than to have a teddy bear item that isn't necessarily a bear.

Reply:
I will trust your judgment on this and revert these changes in a bit once I figure out whether the obsoletion doesn't cause errors